src/common/components/Layout/components/Head/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { Helmet } from 'react-helmet-async';
+
+const Head = () => {
+  const isMakers = import.meta.env.VITE_TEAM_NAME.includes('makers');
```

Comment:
```suggestion
  const isMakers = import.meta.env.MODE === 'makers';
```
mode를 makers로 설정해줬기 때문에 이를 적용하면 좋을 거 같아요
환경변수 하나 더 추가하기 보다요!

물론 cloudflare에서 build 설정을 yarn build:makers, yarn dev:makers로 바꿔야 합니다!

Reply:
아하 너무 좋은 것 같습니다 ~~!!!! 
 
대신 MODE를 사용할 경우 로컬에서도 항상 yarn dev가 아닌 yarn dev:makers 이렇게 해주어야 하는데 
그럼 차라리 저희가 .env (로컬 환경변수)를 메이커스 버전으로 다 사용하기로 했으니까 
아예 디폴트 yarn dev 실행을 vite --mode makers 로 실행시키는 것은 어떠신가요?

Reply:
아항 좋은 거 같습니다~~ 그건 [여기서](https://github.com/sopt-makers/sopt-recruiting-frontend/pull/448) 수정할게요 :)

Reply:
완전 좋습니다~ ~ 주용오빠 확인하면 머지할게요 !

Reply:
넹~ develop에 merge 해주세여 :)

Reply:
아 그리고 `const isMakers = import.meta.env.MODE === 'makers'` 로 바꾸는 건 여기서 수정 부탁드려요~~~
[여기선](https://github.com/sopt-makers/sopt-recruiting-frontend/pull/448) 빌드 명령어를 수정할게요 :)